website/docs/docs/deploy/continuous-integration.md

```diff
@@ -44,12 +40,35 @@ Below describes the conditions when CI checks are run concurrently and when they
 - CI runs with the _same_ PR number and _different_ commit SHAs execute serially because they’re building into the same schema. dbt Cloud will run the latest commit and cancel any older, stale commits. For details, refer to [Smart cancellation of stale builds](#smart-cancellation). 
 - CI runs with the same PR number and same commit SHA, originating from different dbt Cloud projects will execute jobs concurrently. This can happen when two CI jobs are set up in different dbt Cloud projects that share the same dbt repository.
 
-### Smart cancellation of stale builds {#smart-cancellation}
+</Expandable>
+
+<Expandable alt_header="Smart cancellation of stale builds">
 
 When you push a new commit to a PR, dbt Cloud enqueues a new CI run for the latest commit and cancels any CI run that is (now) stale and still in flight. This can happen when you’re pushing new commits while a CI build is still in process and not yet done. By cancelling runs in a safe and deliberate way, dbt Cloud helps improve productivity and reduce data platform spend on wasteful CI runs.
 
 <Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/example-smart-cancel-job.png" width="70%" title="Example of an automatically canceled run"/>
 
-### Run slot treatment <Lifecycle status="team,enterprise" />
+</Expandable>
+
+<Expandable alt_header="Run slot treatment" lifecycle="team,enterprise">
 
 CI runs don't consume run slots. This guarantees a CI check will never block a production run.
+
+</Expandable>
+
+<Expandable alt_header="Compare changes" lifecycle="beta" >
+
+ When a pull request is opened or new commits are pushed, dbt Cloud compares the changes between the last applied state of the production environment (defaulting to deferral for lower computation costs) and the latest changes from the pull request for CI jobs that have the **Run compare changes** option enabled. By analyzing these comparisons, you can gain a better understanding of how the data changes are affected by code changes to help ensure you always ship the correct changes to production and create trusted data products.
+
+ :::info Beta feature
```

Comment:
Might want to remove this here unless the plan is to come back and update this callout. It we choose to leave it, it might be better served at the top of the expandable paragraph

Reply:
good shout! we're planning on removing this 👍